File: api/app/draft_groups.py
"""Groups that use temporary draft collages (one-day capture mode)."""
from __future__ import annotations

from uuid import UUID

REAL_PHOTOS_GROUP_ID = UUID("721bf726-cdda-4ca8-bf22-f345ca0f677b")
DEFECT_PHOTOS_GROUP_ID = UUID("edce2987-daae-4339-8330-8cb96ad912bf")

DRAFT_GROUP_IDS: frozenset[UUID] = frozenset({REAL_PHOTOS_GROUP_ID, DEFECT_PHOTOS_GROUP_ID})


def is_draft_group(group_id: UUID) -> bool:
    return group_id in DRAFT_GROUP_IDS

File: web/lib/draftGroups.ts
export const REAL_PHOTOS_GROUP_ID = "721bf726-cdda-4ca8-bf22-f345ca0f677b";
export const DEFECT_PHOTOS_GROUP_ID = "edce2987-daae-4339-8330-8cb96ad912bf";

const DRAFT_GROUP_IDS = new Set([REAL_PHOTOS_GROUP_ID, DEFECT_PHOTOS_GROUP_ID]);

export function isDraftGroup(groupId: string): boolean {
  return DRAFT_GROUP_IDS.has(groupId);
}
